lua/presence/init.lua

```diff
@@ -502,60 +502,87 @@ function Presence:update_for_buffer(buffer, should_debounce)
         state = status_text,
         assets = assets,
         timestamps = {
-            start = activity_set_at,
+            start = relative_activity_set_at,
         },
     }
 
-    self.log:debug(string.format("Getting project name for %s...", parent_dirpath))
-    local workspace_text = self.options.workspace_text
-    local project_name, project_path = self:get_project_name(parent_dirpath)
+    -- Get the current line number and line count if the user has set the enable_line_number option
+    if self.options.enable_line_number then
```

Comment:
Wait oops my bad lol I got it backwards -_- I actually coalesce Lua boolean options to `0` and `1`s to stay consistent with Vim options, so it looks like we'd need to check for `self.options.enable_line_number == 1`.
```suggestion
    if self.options.enable_line_number == 1 then
```

Reply:
Hah, I had a feeling I did that in the original PR for a reason :P